samcli/lib/hook/exceptions.py

```diff
@@ -20,3 +20,7 @@ class InvalidHookPackageConfigException(UserException):
 
 class PrepareHookException(UserException):
     pass
+
+
+class InvalidSamMetadataProperties(UserException):
```

Comment:
Should we always use Exception or Error as suffix for an exception class?

Reply:
I will update its name